assets/src/dashboard/components/pageStructure/navigationComponents.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+export const LeftRailContainer = styled.nav`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  width: 15%;
+  min-width: 190px;
+  background: ${({ theme }) => theme.colors.white};
+  border-right: ${({ theme }) => theme.leftRail.border};
+`;
```

Comment:
See comment above, may be nice to make this fixed and overlay the app content. could even declare this in the theme:
```
leftRail: {
  width: 'max(15%, 190px)'
}
```
and we could use it for the left-padding on the app content and the width here and avoid a `min-width` property declaration

Reply:
We can break that up to be pure values for the theme